File: app/src/main/java/com/fishweeb/practical/HighScore.java
package com.fishweeb.practical;

public class HighScore
{
    private int Score;
    private String Name;

    HighScore(int _score,String _name)
    {
        Score = _score;
        Name = _name;
    }

    void SetScore(int _score)
    {
        Score = _score;
    }

    void SetName(String _name)
    {
        Name = _name;
    }

    int GetScore()
    {
        return Score;
    }

    String GetName()
    {
        return Name;
    }
}

File: app/src/main/java/edu/androidgroupproject/HighScore.kt
package edu.androidgroupproject

class HighScore
internal constructor(private var Score: Int, private var Name: String) {
    fun SetScore(_score: Int) {
        Score = _score
    }

    fun SetName(_name: String) {
        Name = _name
    }

    fun GetScore(): Int {
        return Score
    }

    fun GetName(): String {
        return Name
    }
}
